src/Coordination.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+
+class Coordination {
+    //Global variables
+    private Computation engine;
+    private List<Integer> in;
+    private List<Integer> out;
+    private ComputeEngineAPI computAPI;
+    private DataToFromComputeAPI dataAPI;
+    private int key;
+
+    //Constructors
+    public Coordination(ComputeEngineAPI computeAPI) {
+        this.computAPI=computeAPI;
+    }
+    public Coordination() {
+        computeAPI = new ComputeEngineAPI();

```

Comment:
I see you used "computAPI" here, wanted to clarify if there is a spelling error or if this was intended to be named as such. If so, consider renaming for clarity

Reply:
On line 14 (oops) ^^

Reply:
I see the confusion, I fixed both the error to be computeAPI, thank you! Should be good now if you want to re-review